specification/marketplaceordering/resource-manager/readme.md

```diff
@@ -119,12 +119,27 @@ python:
   clear-output-folder: true
   package-version: 0.1.0
 ```
+
+``` yaml $(python) && $(track2)
+python-mode: create
+azure-arm: true
+license-header: MICROSOFT_MIT_NO_VERSION
+namespace: azure.mgmt.marketplaceordering
+package-name: azure-mgmt-marketplaceordering
+clear-output-folder: true
+package-version: 1.0.0b1
+```
+
 ``` yaml $(python) && $(python-mode) == 'update'
```

Comment:
Hi, 

Shall we add track2 config to it's own area? something like below, this will make autorest.az onboarding easily.

``` yaml $(python) && $(python-mode) == 'update'
python:
  no-namespace-folders: true
  output-folder: $(python-sdks-folder)/marketplaceordering/azure-mgmt-marketplaceordering/azure/mgmt/marketplaceordering
```

``` yaml $(python) && $(python-mode) == 'update' && $(track2)
no-namespace-folders: true
output-folder: $(python-sdks-folder)/marketplaceordering/azure-mgmt-marketplaceordering/azure/mgmt/marketplaceordering
```

``` yaml $(python) && $(python-mode) == 'create'
python:
  basic-setup-py: true
  output-folder: $(python-sdks-folder)/marketplaceordering/azure-mgmt-marketplaceordering
```

``` yaml $(python) && $(python-mode) == 'create' && $(track2)
basic-setup-py: true
output-folder: $(python-sdks-folder)/marketplaceordering/azure-mgmt-marketplaceordering
```

CC to @msyyc @jsntcy